weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/tabularListViews/columnBuilder.tsx

```diff
@@ -318,7 +318,7 @@ export const buildDynamicColumns = <T extends GridValidRowModel>(
               ) : (
                 <CustomWeaveTypeProjectContext.Provider
                   value={{entity, project}}>
-                  <CellValue value={val} />
+                  <CellValue value={val} field={key.split('.').slice(-1)[0]} />
```

Comment:
Clearer to use `key.split('.').pop()` or `key.substring(key.lastIndexOf('.') + 1)`